E2E-tests/config/substrate/local_nodes.json

```diff
@@ -5,7 +5,7 @@
         "network": "--testnet-magic 42",
         "epoch_length": 120,
         "slot_length": 1,
-        "active_slots_coeff": 0.8,
+        "active_slots_coeff": 0.4,
```

Comment:
Before this was 0.8, but effectively the node was producing like this setting is approx 0.5 because of chain misconfiguration. This change doesn't really change how the chain behaves and is equal to what PC nodes have in their configuration